### PR TITLE
corrected (dead-)links

### PR DESCRIPTION
**Pull request by PhilPhonic**
What does this implement/fix? Explain your changes.
---------------------------------------------------
updated (dead-)links

Does this close any currently open issues?
------------------------------------------
no

Any relevant logs, error output, etc?
-------------------------------------
n/a

Where has this been tested?
---------------------------
**Server (please complete the following information):**
 - OS: debian 11.8
 - PHP Version: 7.4

**Smartphone (please complete the following information):**
 - Mail App [e.g. GMail, Apple Mail]

### CODE REVIEW

File: src/lib/core/zpush.php [hunk 1]
```diff
@@ -825,10 +825,10 @@ static public function PrintZPushLegal($message = "", $additionalMessage = "") {
         $message $additionalMessage
         <br><br>
         More information about Z-Push can be found at:<br>
-        <a href="http://z-push.org/">Z-Push homepage</a><br>
-        <a href="http://z-push.org/download">Z-Push download page</a><br>
-        <a href="https://jira.z-hub.io/browse/ZP">Z-Push Bugtracker</a><br>
-        <a href="https://wiki.z-hub.io/display/ZP">Z-Push Wiki</a> and <a href="https://wiki.z-hub.io/display/ZP/Roadmap">Roadmap</a><br>
+        <a href="https://z-push.org/">Z-Push homepage</a><br>
```
Comment (fbartels) [inline src/lib/core/zpush.php]: The current redirect does not seem to allow this. for me this throws a 403.

File: src/lib/core/zpush.php [hunk 2]
```diff
@@ -826,9 +826,9 @@ static public function PrintZPushLegal($message = "", $additionalMessage = "") {
         <br><br>
         More information about Z-Push can be found at:<br>
         <a href="http://z-push.org/">Z-Push homepage</a><br>
-        <a href="http://z-push.org/download">Z-Push download page</a><br>
```
Comment (matidau) [inline src/lib/core/zpush.php:829]: @PhilPhonic We (thanks to @fbartels) now have z-push.org working against the github pages site.
Can you revert this change to `https://z-push.org/download` or `https://z-push.org/download/`

Comment (matidau) [reply src/lib/core/zpush.php:829]: sorry http or https is fine 🙂

**Review by matidau — CHANGES_REQUESTED** [review]

**Comment by matidau** [comment]
Thanks for this 😊

**Pull request merged**